activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/pom.xml

```diff
@@ -171,17 +155,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>repackage</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
With this plugin, we can run the app with below command. Why removing it here?

java -jar target\azure-ad-integration-spring-boot-autoconfigure-sample-0.0.1-SNAPSHOT.jar

Reply:
@snicoll BTW, could you sign the CLA? :)

Reply:
Idem. Remove it and you'll see it's still generated.

Reply:
Yes, it's generated, but it would have below error.

no main manifest attribute, in target\azure-ad-integration-spring-boot-autoconfigure-sample-0.0.1-SNAPSHOT.jar

Reply:
uh?

Reply:
https://docs.spring.io/spring-boot/docs/current/maven-plugin/repackage-mojo.html